controllers/account/api/v1/accountuser_types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// AccountUserSpec defines the desired state of AccountUser
+type AccountUserSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Foo is an example field of AccountUser. Edit accountuser_types.go to remove/update
+	Foo string `json:"foo,omitempty"`
```

Comment:
Empty `AccountUser`?

Reply:
AccountUser's function  is watch user, when user create will create the account

Reply:
if not do this，watching user's logic should add in account,accoun need to watch payment,account and user,
too complicated

Reply:
Got, maybe replace `foo` with `status` or other meaningful var.

Reply:
oh，no meteringful var is need，i will delete all meteringless code

Reply:
done